yogstation/code/game/gamemodes/battle_royale/battleroyale.dm

```diff
@@ -0,0 +1,211 @@
+GLOBAL_VAR(thebattlebus)
+GLOBAL_LIST_EMPTY(battleroyale_players) //reduce iteration cost
+
+/datum/game_mode/fortnite
+	name = "battle royale"
+	config_tag = "battleroyale"
+	report_type = "battleroyale"
+	false_report_weight = 0
+	required_players = 1 //Everyone is an antag in this mode
+	required_enemies = 1
+	recommended_enemies = 1
+	antag_flag = ROLE_BATTLEROYALE
+	enemy_minimum_age = 0
+
+	announce_span = "warning"
+	announce_text = "Attention ALL space station 13 crewmembers,\n\
+	<span class='danger'><b>John wick</b></span> is in grave danger and he NEEDS your help to wipe all the squads in the tilted towers. To do this, he'll need\n\
```

Comment:
perhaps change fortnite specific things to ss13 stuff, e.g. change john wick to george melons